### PR TITLE
fix cstyle issue in ./src/istgt_lu.h

### PR DESCRIPTION
**Pull request by cbanupama**

### CODE REVIEW

**Comment by utkarshmani1997** [comment]
@cbanupama it will be good if you provide the output of the `./cstyle.pl filename`.

File: src/istgt_lu.h [hunk 1]
```diff
@@ -138,27 +139,31 @@ typedef enum {
  */
 #define ISTGT_MAX_LU_RUNNING_STATE_RETRY_COUNT 45
 
-typedef enum {
+typedef enum
```
Comment (vishnuitta) [inline src/istgt_lu.h:142]: these changes are not required as cstyle is not showing any error here, and, in the code, 'flower braces' should be given in the same line of 'enum', 'struct', 'if', 'for' etc.

**Review by vishnuitta — CHANGES_REQUESTED** [review]
As mentioned in other PRs, request you to re-do the PR

**Comment by mittachaitu** [comment]
closing this PR as there were no changes made after providing the comments from 7 Oct 2018. If you get time to rework on the comments you can open the PR for review.

**Pull request closed without merging**